requirements/system/systemmodel/system-model.tex

```diff
@@ -13,9 +13,11 @@ \subsection{System Model}
   \centering
   \includegraphics[width=\linewidth]{./system/systemmodel/images/accessjob.png}
   \caption{This diagram depicts how a user accesses his running job.
-  After starting the job, the CS returns WC ip and container id.
-  Using this information the user can connect to the container executing his job at any time.
-  Only the user that started the job and administrators can access the job.}
```

Comment:
Insert:
The user has a running job on one of the work machines that he wants to connect to.